packages/react-core/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to @patternfly/react-core
```

Comment:
@amarie401  and I had been adding some useful contribution notes to a document.  Maybe we can incorporate them here.  WE had not in setting up work space, running unit test and building and running the docs locally.
https://docs.google.com/document/d/1a5YtDWc8C55ewOdrp6h4SM0fmjGO_1bbne20R0XLS1U/edit

Reply:
Oh nice I forgot all about those parts. I just wanted to make sure to capture the docs piece here. I think it would be good to move that info from the doc into this md file as well. Do you want me to do that in this PR or another?

Reply:
It can be added in another PR.